br/pkg/backup/store.go

```diff
@@ -218,8 +224,10 @@ func startBackup(
 				retry := -1
 				return utils.WithRetry(ectx, func() error {
 					retry += 1
-					logutil.CL(ectx).Info("backup to store", zap.Uint64("storeID", storeID),
-						zap.Int("retry", retry), zap.Int("reqIndex", reqIndex))
+					if retry > 1 {
```

Comment:
```suggestion
					if retry >= 1 {
```

Reply:
actually it's intentional. I think the first retry can be tolerant.